helm/csi-powerstore/templates/controller.yaml

```diff
@@ -272,6 +272,8 @@ spec:
               value: {{ .Values.driverName }}
             - name: X_CSI_POWERSTORE_EXTERNAL_ACCESS
               value: {{ .Values.externalAccess }}
+            - name: X_CSI_NFS_ACLS
+              value: "{{ .Values.nfsAcls }}"
```

Comment:
Please check if default value is needed here as well. Or are we handling empty string passed in?

Reply:
Yes, empty string is handled in code and is default to 0777 mode as before. So not setting it here.